tests/Aspire.Hosting.Docker.Tests/DockerComposePublisherTests.cs

```diff
@@ -36,7 +39,10 @@ public async Task PublishAsync_GeneratesValidDockerComposeFile()
                          .WithReference(cs)
                          .WithArgs("--cs", cs.Resource);
 
-        builder.AddProject<TestProject>("project1", launchProfileName: null)
+        builder.AddProject(
+            "project1",
+            "..\\TestingAppHost1\\TestingAppHost1.MyWebApp\\TestingAppHost1.MyWebApp.csproj",
```

Comment:
Path.Combine?

Reply:
AddProject does it internally correctly relative to the apphost path (which is the test folder in this instance).